apps/test/unit/aiDifferentiation/AiDiffChatTest.jsx

```diff
@@ -0,0 +1,292 @@
+import {render, screen, fireEvent, waitFor} from '@testing-library/react';
+import React from 'react';
+
+import {Role} from '@cdo/apps/aiComponentLibrary/chatMessage/types';
+import AiDiffChat from '@cdo/apps/aiDifferentiation/AiDiffChat';
+import {EVENTS, PLATFORMS} from '@cdo/apps/metrics/AnalyticsConstants';
+import analyticsReporter from '@cdo/apps/metrics/AnalyticsReporter';
+import HttpClient from '@cdo/apps/util/HttpClient';
+import {AiInteractionStatus as Status} from '@cdo/generated-scripts/sharedConstants';
+import i18n from '@cdo/locale';
+
+jest.mock('@react-pdf/renderer', () => {
+  return {
+    PDFDownloadLink: () => null,
+    StyleSheet: {
+      create: () => null,
+    },
+  };
+});
+
+const defaultProps = {
+  closeTutor: () => {},
+  open: true,
+  lessonId: 2,
+  lessonName: 'test_lesson',
+  unitDisplayName: 'test unit name',
+};
+
+const defaultChatResponse = {
+  chat_message_text: "Beep boop I'm a bot",
+  status: Status.OK,
+  session_id: '123abc',
+};
+
+describe('AiDiffChat', () => {
+  let fetchStub;
+  let sendEventSpy;
+
+  beforeEach(() => {
+    window.HTMLElement.prototype.scrollIntoView = () => {};
+    sessionStorage.clear();
+    fetchStub = jest
+      .spyOn(HttpClient, 'post')
+      .mockResolvedValue(
+        Promise.resolve(new Response(JSON.stringify(defaultChatResponse)))
+      );
+
+    sendEventSpy = jest.spyOn(analyticsReporter, 'sendEvent');
+  });
+
+  afterEach(() => {
+    sessionStorage.clear();
+    jest.restoreAllMocks();
+  });
+
+  it('initial message and suggested prompts are rendered', () => {
+    render(<AiDiffChat {...defaultProps} />);
+    const message = screen.getByLabelText(i18n.aiChatMessageBot());
+    expect(message).toHaveTextContent(
+      "Hi! I'm your AI Teaching Assistant. What can I help you with? Here are some things you can ask me."
+    );
+    //suggested prompts
+    expect(screen.getAllByRole('checkbox')).toHaveLength(4);
+    screen.getByRole('checkbox', {name: 'Explain a concept'});
+    screen.getByRole('checkbox', {
+      name: 'Give an example to use with my class',
```

Comment:
I wonder if it makes sense to go ahead a i18n the strings and re-use them here and in SUGGESTED_PROMPTS so we only have to change one place if they get updated? I don't feel strongly about it, though